lesson02/userdata.sh

```diff
@@ -10,5 +10,5 @@ sudo service docker start
 #All commands here are executed as super admin
 #but still, let's add the ec2-user to the docker group
 usermod -a -G docker ec2-user 
- #Run the nginx 
-docker run -d -p 80:80 --restart=always nginx:${docker_tag}
\ No newline at end of file
+#Run the nginx
+docker run -d -p 80:80 --restart=always ${docker_image}:${docker_tag}
```

Comment:
Nice! 👍 
non-related to this PR, since `userdata.sh` is a template file, would be nice adding a `.tpl` at the end of the file name